UndertaleModLib/Decompiler/Instructions/Decompiler.ExpressionTwo.cs

```diff
@@ -34,15 +34,69 @@ public override Statement CleanStatement(DecompileContext context, BlockHLStatem
             return this;
         }
 
-        public override string ToString(DecompileContext context)
+        private string AddParensIfNeeded(Expression argument, DecompileContext context)
+        {
+            string arg = argument.ToString(context);
+            bool needsParens;
+            if (arg.Substring(0, 1) != "(" &&
```

Comment:
Just noticed, why use substring, instead of `arg[0]`

Reply:
Oh, I forgot that was an option.